boards/posix/doc/arch_soc.rst

```diff
@@ -49,14 +49,19 @@ simulation and framework, while being fully decoupled of the host.
 Host system dependencies
 ========================
 
-This port is designed to run in POSIX compatible operating systems,
+This port is designed to run in Unix operating systems,
```

Comment:
Isn't this even more confusing than it already is, because macOS is a descendant of UNIX and is in fact [UNIX certified](https://www.opengroup.org/openbrand/register/)?

Reply:
agree, and Linux is nt Unit, it is Unix-like, and macOS falls under the same category ;-(

https://en.wikipedia.org/wiki/List_of_Unix_systems

Reply:
Fair enough lets be clearer :)